data/json/items/classes/comestible.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "id": "food",
```

Comment:
This was changed in response to @Coolthulhu earlier review:

> Why explicitly specify 1 volume for everything?

It's possible we should specify `volume=1` as the default for `itype`?